dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/MetadataImportExportControllerTest.java

```diff
@@ -394,4 +395,61 @@ void testImportCategoryComboAndCategory() {
     assertNotNull(categories);
     assertFalse(categories.stream().anyMatch(JsonValue::isNull));
   }
+
+  @Test
+  void testImportDashboardWithInvalidLayout_UpdateFlow() {
+    JsonImportSummary createReport =
+        POST("/metadata", WebClient.Body("dashboard/import_dashboard_with_valid_layout.json"))
+            .content(HttpStatus.OK)
+            .get("response")
+            .as(JsonImportSummary.class);
+
+    assertEquals("OK", createReport.getStatus());
+    assertEquals(1, createReport.getStats().getCreated());
+    assertEquals(0, createReport.getStats().getIgnored());
+    assertEquals(0, createReport.getStats().getUpdated());
+    assertEquals(1, createReport.getStats().getTotal());
+
+    JsonImportSummary updateReport =
+        POST("/metadata", WebClient.Body("dashboard/import_dashboard_with_invalid_layout.json"))
+            .content(HttpStatus.CONFLICT)
+            .get("response")
+            .as(JsonImportSummary.class);
+
+    assertEquals("ERROR", updateReport.getStatus());
+    assertEquals(0, updateReport.getStats().getCreated());
+    assertEquals(2, updateReport.getStats().getIgnored());
+    assertEquals(0, updateReport.getStats().getUpdated());
+    assertEquals(2, updateReport.getStats().getTotal());
+  }
+
+  @Test
+  void testImportDashboardWithValidLayout_CreateFlow() {
+    JsonImportSummary report =
+        POST("/metadata", WebClient.Body("dashboard/import_dashboard_with_valid_layout.json"))
+            .content(HttpStatus.OK)
+            .get("response")
+            .as(JsonImportSummary.class);
+
+    assertEquals("OK", report.getStatus());
+    assertEquals(1, report.getStats().getCreated());
+    assertEquals(0, report.getStats().getIgnored());
+    assertEquals(0, report.getStats().getUpdated());
+    assertEquals(1, report.getStats().getTotal());
+  }
+
+  @Test
+  void testImportDashboardWithInvalidLayout_CreateFlow() {
+    JsonImportSummary report =
+        POST("/metadata", WebClient.Body("dashboard/import_dashboard_with_invalid_layout.json"))
+            .content(HttpStatus.CONFLICT)
+            .get("response")
+            .as(JsonImportSummary.class);
+
+    assertEquals("ERROR", report.getStatus());
+    assertEquals(0, report.getStats().getCreated());
+    assertEquals(2, report.getStats().getIgnored());
```

Comment:
This seems to be a separate bug, expected should be 1 not 2. Have created a new issue to follow up on - [DHIS2-15604](https://dhis2.atlassian.net/browse/DHIS2-15604)